test/Microsoft.ML.Tests/OnnxTests.cs

```diff
@@ -282,6 +294,60 @@ public void KeyToVectorWithBagTest()
             Done();
         }
 
+        [Fact]
+        public void WordEmbeddingsTest()
+        {
+            string dataPath = GetDataPath(@"small-sentiment-test.tsv");
+            var pipeline = new Legacy.LearningPipeline(0);
+
+            pipeline.Add(new Legacy.Data.TextLoader(dataPath)
+            {
+                Arguments = new TextLoaderArguments
+                {
+                    Separator = new[] { '\t' },
+                    HasHeader = false,
+                    Column = new []
+                    {
+                        new TextLoaderColumn()
+                        {
+                            Name = "Cat",
+                            Source = new [] { new TextLoaderRange(0, 3) },
+                            Type = Legacy.Data.DataKind.TX
+                        },
+                    }
+                }
+            });
+            
+            var modelPath = GetDataPath(@"shortsentiment.emd");
+            var embed = new WordEmbeddings() { CustomLookupTable = modelPath };
+            embed.AddColumn("Cat", "Cat");
+            pipeline.Add(embed);
+            var model = pipeline.Train<EmbeddingsData, EmbeddingsResult>();
+
+            var subDir = Path.Combine("..", "..", "BaselineOutput", "Common", "Onnx", "WordEmbeddings");
```

Comment:
>.Combine("..", "..", "BaselineOutput", "Common", [](start = 29, length = 48)

GetOutputPath("Onnx") ?

Reply:
Sorry, could you clarify this a little bit more? Are you saying I should just have these files in the default TestOutput directory? All of the other ONNX tests have the test files in this directory that I used.

---
In reply to: [226402328](https://github.com/dotnet/machinelearning/pull/1249#discussion_r226402328) [](ancestors = 226402328)